src/components/Upload.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import PropTypes from 'prop-types';
+
+const Upload = ({ onDrop, children, ...props }) =>
+  (<Dropzone
+    {...props}
+    // What do we accept as valid?
+    accept="image/jpeg, image/png"
+    onDrop={onDrop}
+  >
+    {children}
+  </Dropzone>);
+
+Upload.propTypes = {
+  onDrop: PropTypes.func.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+export default Upload;
```

Comment:
Ok. We should give it a style then that fits with the rest of the design. It should have a preview of the image, etc.

It also needs it wrapped in a Field from redux forms, so that we can include it in a form. The value of the component can be stored as a [DataURI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs).